source/jormungandr/jormungandr/realtime_schedule/xml_parser.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2001-2016, Canal TP and/or its affiliates. All rights reserved.
```

Comment:
I think this file should be named by the connector (it will there that we put the connector no ?)